nguyen_dinh_hieu/laravel-fw/app/Http/Controllers/shopController.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Http\Controllers;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Http\Request;
+
+class shopController extends Controller
+{
+    public function shop()
+    {
+        $products = DB::table('product')
+        ->orderByDesc('price')
+        ->paginate(6);
```

Comment:
```suggestion
            ->orderByDesc('price')
            ->paginate(6);
```
số 6 e lấy từ config nhé